client/packages/invoices/src/InboundShipment/DetailView/modals/InboundLineEdit/InboundLineEdit.tsx

```diff
@@ -163,15 +163,12 @@ export const InboundLineEdit: FC<InboundLineEditProps> = ({
         }
         height={height}
         width={1024}
+        enableAutocomplete /* Required for previously entered batches to be remembered and suggested in future shipments */
       >
         {isLoading ? (
           <BasicSpinner messageKey="saving" />
         ) : (
-          <form
-            autoComplete={
-              'on' /* Required for previously entered batches to be remembered and suggested in future shipments */
-            }
-          >
+          <>
```

Comment:
Can't have a <form> within a <form>, so added optional autocomplete prop for this case

Reply:
yes, nicely handled